src/pymmcore_plus_sandbox/__init__.py

```diff
@@ -432,6 +446,91 @@ def _on_mda_finished(self, sequence: MDASequence) -> None:
         pass
 
 
+class GridPlanTensorStoreHandler(TensorStoreHandler):
+    def __init__(
+        self,
+        sequence: MDASequence,
```

Comment:
i made a point of resisting passing a sequence to the handler constructor, with the idea being that handler should be able to deal with multiple sequences (we don't necessarily imply a 1-to-1 mapping of sequence-handler).  You can access the currently running sequence via the public `TensorStoreHandler.current_sequence` (however, I'm realizing now that it's poorly implemented since it will be an attribute error prior to `sequenceStarted` being called).